src/customisations/components/views/messages/MAudioBodyWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+
+import { IBodyProps } from "../../../../components/views/messages/IBodyProps";
+import { default as OriginalMAudioBody } from "../../../../components/views/messages/MAudioBody";
+
+/**
+ * This component wraps MAudioBody to provide a stable customisation entry point.
+ */
```

Comment:
What additional stability is gained over just using https://github.com/vector-im/element-web/blob/develop/docs/customisations.md#custom-components

Reply:
Maybe „stable“ in the comment was not the best choice :slightly_smiling_face: